airflow/providers/trino/hooks/trino.py

```diff
@@ -95,7 +95,12 @@ def get_conn(self) -> Connection:
         elif db.password:
             auth = trino.auth.BasicAuthentication(db.login, db.password)  # type: ignore[attr-defined]
         elif extra.get("auth") == "jwt":
-            auth = trino.auth.JWTAuthentication(token=extra.get("jwt__token"))
+            if("jwt__file" in extra):
+                with open(extra.get("jwt__file"), 'r') as jwt_file:
```

Comment:
```suggestion
            if "jwt__file" in extra:
                with open(extra["jwt__file"], "r") as jwt_file:
```

Reply:
I'm hesitant to use the bracket notation `extra["..."]` in your suggestion because `extra.get("...")` is used elsewhere in this file before this commit, and I wanted to be consistent with the existing notation. Seems to be used in the code for other providers/hooks, too. I will, however, put in the other suggested changes from the static check